### PR TITLE
Add hab bldr encrypt command

### PR DESCRIPTION
**Pull request by chefsalim**
This change adds a 'hab bldr encrypt' command to the cli. The command takes plain text and encrypts it with the builder's public key. The encrypted text can only be decrypted by the builder, and can be used to pass secrets to the builder as needed for CI functionality.

Signed-off-by: Salim Alam <salam@chef.io>

tenor-42487944

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/hab/src/cli.rs [hunk 1]
```diff
@@ -75,6 +75,18 @@ pub fn get() -> App<'static, 'static> {
                     "Ring key name, which will encrypt communication messages")
             )
         )
+        (@subcommand bldr =>
```
Comment (reset) [inline components/hab/src/cli.rs]: This is great, we might want to consider moving the `job` sub command under here at some point

**Comment by reset** [comment]
@thesentinels approve

tenor-193339435

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/267511052).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/267511052).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**